### PR TITLE
Standard side sheet

### PR DESCRIPTION
**Pull request by ingridklepsvik**

### CODE REVIEW

File: src/components/general/SideSheet/Standard/styles.less [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+        height: 100%;
+        left: 0;
+        top: 0;
+        bottom: 0;
+        right: 0;
+        transition: transform .1s ease-out;
+     
+        pointer-events: all;
+
+        /* &.isCollapsed {
```
Comment (eskil0312) [inline src/components/general/SideSheet/Standard/styles.less]: Remove comment?

File: src/components/general/SideSheet/Standard/index.tsx [hunk 2]
```diff
@@ -0,0 +1,107 @@
...
+        <div className={containerClassNames} key={id}>
+            <div className={styles.header}>
+                <div className={styles.collapseButtonContainer}>
+                    <CollapseExpandButton
+                        isOpen={isOpen}
+                        onClick={toggleOpen}
+                        screenPlacement={screenPlacement}
+                    />
+                </div>
+                {isOpen && title && <div>{title}</div>}
```
Comment (eskil0312) [inline src/components/general/SideSheet/Standard/index.tsx]: Maybe the title should have som styling?

File: src/components/general/SideSheet/Standard/styles.less [hunk 3]
```diff
@@ -0,0 +1,61 @@
...
+        } */
+    }
+
+    .header{
+        display: flex;
+        align-items: center;
+        overflow: hidden;
+    }
+
+    .content {
```
Comment (eskil0312) [inline src/components/general/SideSheet/Standard/styles.less]: Maybe the css should reflect the html structure? .content ... { .header ... { .collapseButtonContainer ... }}}

File: src/components/general/SideSheet/Standard/index.tsx [hunk 4]
```diff
@@ -0,0 +1,107 @@
...
+        '--mobile-max-width'
+    );
+
+    if (isOpen && windowWidth < parseInt(mobileWidth)) {
+        return (
+            <OverlayPortal show={isOpen}>
+                <Scrim show={isOpen}>
+                    <SideSheet id={id} title={title} isOpen={isOpen} onOpenChange={onOpenChange}>
+                        {children}
+                    </SideSheet>
```
[the remark this answers is not in the document]
Comment (ingridklepsvik) [reply src/components/general/SideSheet/Standard/index.tsx]: Moved into a const

File: src/components/general/SideSheet/Standard/styles.less (hunk 1 above)
Comment (ingridklepsvik) [reply src/components/general/SideSheet/Standard/styles.less]: Comment removed

File: src/components/general/SideSheet/Standard/index.tsx (hunk 2 above)
Comment (ingridklepsvik) [reply src/components/general/SideSheet/Standard/index.tsx]: Styled title

File: src/components/general/SideSheet/Standard/styles.less (hunk 3 above)
Comment (ingridklepsvik) [reply src/components/general/SideSheet/Standard/styles.less]: Fixed

File: src/components/general/SideSheet/Standard/index.tsx [hunk 5]
```diff
@@ -0,0 +1,107 @@
...
+    }, [rootElement, windowWidth]);
+
+    const mobileWidth = getComputedStyle(document.documentElement).getPropertyValue(
+        '--mobile-max-width'
+    );
+
+    if (isOpen && windowWidth < parseInt(mobileWidth)) {
+        return (
+            <OverlayPortal show={isOpen}>
+                <Scrim show={isOpen}>
```
[the remark this answers is not in the document]
Comment (ingridklepsvik) [reply src/components/general/SideSheet/Standard/index.tsx]: Romoved

File: src/components/general/SideSheet/Standard/index.tsx [hunk 6]
```diff
@@ -32,6 +32,7 @@ const SideSheet: React.FC<StandardSideSheetProps> = ({
         useComponentDisplayClassNames(styles),
         {
             [styles.isCollapsed]: !isOpen,
+            [styles.isOpen]: isOpen,
```
Comment (eskil0312) [inline src/components/general/SideSheet/Standard/index.tsx]: Do we need specific styling for open and collapsed? There are only two states, if it is not collapsed then we know it is open.

Comment (ingridklepsvik) [reply src/components/general/SideSheet/Standard/index.tsx]: Romoved one of the states

**Pull request merged**